miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/util/form/TagUtils.java

```diff
@@ -54,7 +54,8 @@ private static Authentication getAuthentication() {
 
   public static String docsVersion() {
     if (Version.VERSION.matches("^\\d*\\.\\d*\\.\\d*$")) {
-      return Version.VERSION.substring(0, Version.VERSION.lastIndexOf(".")) + ".x";
+      //FIXME once we know how readthedocs handles version tags
+      return Version.VERSION;
```

Comment:
It seems like it'll build versions that match our tag names, like "v0.2.190" and create a "stable" version for our most recent release, and "latest" pointing to current develop branch. Probably safest to just return "latest" for now so we know it won't be a broken link in the next release though?